src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCPersonalAccessTokenManagementService.java

```diff
@@ -0,0 +1,43 @@
+package de.tum.in.www1.artemis.service.connectors.localvc;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_LOCALVC;
+
+import java.security.SecureRandom;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.web.rest.UserResource;
+
+@Service
+@Profile(PROFILE_LOCALVC)
+public class LocalVCPersonalAccessTokenManagementService {
+
+    private static final Logger log = LoggerFactory.getLogger(UserResource.class);
+
+    @Value("${artemis.version-control.version-control-access-token:#{false}}")
```

Comment:
Is this correct? shouldn't it rather be:
```suggestion
    @Value("${artemis.version-control.version-control-access-token:false}")
```

Reply:
semantically this does not make any difference in this case, I guess your proposal is more concise